skimage/_shared/fast_exp.h

```diff
@@ -3,15 +3,18 @@
  * Reference [2]: https://doi.org/10.1162/089976600300015033
  * Additional improvements by Leonid Bloch. */
 
+#include <stdint.h>
+
 /* use just EXP_A = 1512775 for integer version, to avoid FP calculations */
-#define EXP_A (1512775.3951951856938)  /* 2^20*ln2 */
+#define EXP_A (1512775.3951951856938)  /* 2^20÷ln2 */
```

Comment:
Thanks for the fix! But can you please separate it to a different commit? It has nothing to do with introducing a single precision version. Also, maybe `/` instead of `÷` - non ASCII characters can cause problems in some scenarios. Nothing against using them when needed, but here a simple `/` will suffice.

Reply:
`/` will of course suffices :-)